helm_deploy/manage-hmpps-auth-accounts/templates/ingress.yaml

```diff
@@ -12,6 +12,11 @@ metadata:
     heritage: {{ .Release.Service }}
   annotations:
     kubernetes.io/ingress.class: "nginx"
+    # note this limit is per ingress replica
+    nginx.ingress.kubernetes.io/limit-rpm: 10
+    # setting to 429 (default is 503) so we can alert on this
```

Comment:
default is 503 :rofl: If only there was an http status code called something like 429=Too Many Requests.